search/src/main/scala/uk/ac/wellcome/platform/api/search/swagger/SwaggerDocs.scala

```diff
@@ -15,28 +15,25 @@ import io.swagger.v3.oas.annotations.enums.ParameterIn
 import io.swagger.v3.oas.annotations.media.{Content, Schema}
 import javax.ws.rs.{GET, Path}
 import grizzled.slf4j.Logging
-import uk.ac.wellcome.api.display.models.{
-  ApiVersions,
-  DisplayImage,
-  DisplayWork
-}
+import uk.ac.wellcome.api.display.models.{DisplayImage, DisplayWork}
 import uk.ac.wellcome.platform.api.models.ApiConfig
 import uk.ac.wellcome.platform.api.search.models._
 import uk.ac.wellcome.platform.api.search.rest.DisplayResultList
 import weco.http.models.DisplayError
 
 class SwaggerDocs(apiConfig: ApiConfig) extends Logging {
-
-  val version = ApiVersions.v2
-
-  val url =
-    s"${apiConfig.scheme}://${apiConfig.host}/${apiConfig.pathPrefix}/$version/"
+  val url = apiConfig.publicRootPath match {
+    case "" =>
+      s"${apiConfig.publicScheme}://${apiConfig.publicHost}/"
+    case path =>
+      s"${apiConfig.publicScheme}://${apiConfig.publicHost}$path/"
```

Comment:
I'm not clear why you're matching here, when dropping `${apiConfig.publicRootPath}` into the string would have the same effect.

Reply:
Oops, think this was from an earlier iteration